www/styles/Chip.module.css

```diff
@@ -26,3 +27,10 @@
   cursor: pointer;
   opacity: .8;
 }
+
+
+.icons {
+  color: var(--skyBlue);
+  display: inline-block;
+  margin-left: .5rem;
```

Comment:
I don't like using `rem` as a unit because if we change the global font size it changes this. Its kind of controversial though.

Reply:
are you not the one who did this lmao